src/factoids.coffee

```diff
@@ -43,22 +43,22 @@ module.exports = (robot) ->
       to ?= msg.message.user.name
       msg.send "#{to.trim()}: #{fact.value}"
 
-  robot.respond new RegExp("[#{prefix}]([\\w\\s-]{2,}\\w)", 'i'), (msg) =>
+  robot.respond new RegExp("[#{prefix}]([\\w\\s-]{1,}\\w)", 'i'), (msg) =>
```

Comment:
I don't think changing this regex (and the one above) should be necessary. Have you tested it without this change?